src/Shared/UnitTests/FileMatcher_Tests.cs

```diff
@@ -2112,7 +2112,7 @@ private void GetMatchingDirectories(string[] candidates, string path, string pat
                             int nextSlash = normalizedCandidate.IndexOfAny(FileMatcher.directorySeparatorCharacters, path.Length + 1);
                             if (nextSlash != -1)
                             {
-                                
+
```

Comment:
there shouldn't be an empty line here at all

Reply:
```suggestion
```